## Common Settings

# Booleans
profiling_skip: true #skip profiling?
profiling_skip_load_map: true #if profiling was skipped, load a preexisting map?
profiling_fill_octomap: true  #Create an octomap in the initial scans?
profiling_fill_octomap_continuously: false
profiling_use_symmetry: true

# Other
profiling_scan_speed: 1.0
profiling_method: 1 # 0 for adaptive circular, 1 for bounded box


## Adaptive Circular
profiling_circular_waypoints: 5
